src/test/java/uk/gov/companieshouse/api/accounts/validation/IntangibleAssetsValidatorTest.java

```diff
@@ -372,6 +373,28 @@ void multipleYearFilerNoTotalFieldsMatch() throws ServiceException, DataExceptio
         assertTrue(errors.containsError(createError(INCORRECT_TOTAL, "$.intangible_assets.total.cost.at_period_end")));
     }
 
+    @Test
```

Comment:
Can a unit test for an incorrect amortisation at period end be added, please?

Reply:
Added a unit test to cover incorrect amortisation at period end